client/server/middleware/unsupported-browser.js

```diff
@@ -1,15 +1,9 @@
 /**
  * Internal dependencies
  */
-import config from '@automattic/calypso-config';
 import { addQueryArgs } from 'calypso/lib/url';
 
 export default () => ( req, res, next ) => {
-	if ( ! config.isEnabled( 'redirect-fallback-browsers' ) ) {
```

Comment:
I'd suggest we set 'redirect-fallback-browsers:true` in prod environments first, and after having it for a few days without incidents, then delete the flags.

Reply:
Done!